src/pages/self-assessment/self-assessment.html

```diff
@@ -46,7 +46,9 @@ <h1 id="category">{{area.title}}</h1>
   </ion-row> -->
 
   <ion-grid>
-    <h1 text-center> DRAG SLIDER TO RATE FROM 0 - 10 HOW WELL YOU ARE DOING IN EACH CATEGORY</h1>
+    <!-- <h1 text-center> DRAG SLIDER TO RATE FROM 0 - 10 HOW WELL YOU ARE DOING IN EACH CATEGORY</h1> -->
+    <h1 id="self-eval">Self evaluation</h1>
```

Comment:
I think we should avoid using ID's if we don't need to. It causes specificity issues down the line.

Reply:
Ok, I will change that now.